dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs

```diff
@@ -0,0 +1,37 @@
+using Xunit;
```

Comment:
Alter the Teste project folder name 
from:
{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs
to:
{{cookiecutter.project_name}}/test/project.Tests/FunctionTest.cs

Reply:
fixed